pymc/smc/smc.py

```diff
@@ -135,16 +135,16 @@ def __init__(
 
```

Comment:
This docstring is lacking the [short summary](https://pymc-data-umbrella.xyz/en/latest/sprint/tutorials/docstring_tutorial.html#short-summary). You can add "Initialize the SMC_kernel class" for now. Otherwise the summary previews look bad:

![imatge](https://user-images.githubusercontent.com/23738400/195640893-0b3e7cdf-d0d8-4079-b43a-5239616e25f9.png)